test/lib/module_test.rb

```diff
@@ -14,12 +14,22 @@ def initialize(info = {})
       @failures = 0
       @skipped = 0
       super
+
+      register_options(
+        [
+          OptString.new("TestName", [false, "Run a specified test method name.", nil]),
+        ], self.class
+      )
     end
 
     def run_all_tests
-      tests = self.methods.select { |m| m.to_s =~ /^test_/ }
+      tests = datastore['TestName'].present? ? [datastore['TestName'].to_sym] : self.methods.select { |m| m.to_s =~ /^test_/ }
```

Comment:
This could probably be `datastore['TestNames'].split(",").map(&:to_sym)` or something to support running multiple tests, but I'll skip that for now as I've only needed to run one test manually at at time